docs-chef-io/content/server/v14/config_rb_server_optional_settings.md

```diff
@@ -724,7 +724,7 @@ This configuration file has the following settings for `nginx`:
     Starting with Chef Infra Server 14.3, this value defaults to `'TLSv1.2'` for
     enhanced security. Previous releases defaulted to `'TLSv1 TLSv1.1 TLSv1.2'`,
     which allowed for less secure SSL connections. TLS 1.2 is supported on
-    Chef Infra Client 10.16.4 and later on Linux, Unix, and macOS, and on Chef 
+    Chef Infra Client 10.16.4 and later on Linux, Unix, and macOS, and on Chef
     Infra Client 12.8 and later on Windows. If it is necessary to support these older end-of-life
```

Comment:
```suggestion
    Infra Infra Client 12.8 and later on Windows. If it is necessary to support these older end-of-life
```

Reply:
I wasn't sure what to tweak on this one -- looks like it does say Chef Infra Client right now, just split across lines here -- did you want to have "Chef Infra Client" together on one line?